ParkingZoneApp/Areas/Admin/Views/ParkingZone/Index.cshtml

```diff
@@ -35,6 +35,11 @@
                                     <i class="fas fa-trash-alt"></i> Delete
                                 </a>
                             </li>
+                            <li class="nav-item" style="margin-left: 10px;">
+                                <a asp-action="CurrentCars" asp-route-id="@item.Id" class="btn btn-outline-success" tabindex="-1" aria-disabled="true">
+                                    <i class="fas fa-car"></i> Current Cars
```

Comment:
```suggestion
                                    Current Cars
```
you are not using it right

Reply:
Actually that i tag is for icon. Can you clearify what Im using wrong

Reply:
sorry I thought it is Italics tag